README.md

```diff
@@ -0,0 +1,11 @@
+# How to preview the documentation changes locally?
+
+The documentation can hosted on a local webserver via httpd.
+
+You can run it with the following command, note that it requires docker:
+
+```shell
+sh start-preview.sh
+```
+
+Then you can open localhost:8080 on your browser and browse the documentation
```

Comment:
Can we make a link for localhost:8080?
Also let's add a full stop at the end of the sentence